templates/static/form_handler.js

```diff
@@ -89,7 +111,7 @@ document.getElementById('url-form').addEventListener('submit', async function(ev
                 downloadButton.addEventListener('click', () => {
                     headline.style.pointerEvents = 'none'; // Disable pointer events on the headline
                     const imageElement = container.querySelector('img');
-                    downloadImage(imageElement, headline.textContent, headline, fontSizeInput.value, colorInput.value);
+                    downloadImage(imageElement, headline.textContent, headline, fontSizeInput.value, colorInput.value, fontTypeInput.value);
```

Comment:
I get 422 unprocessable entity when trying to download an image:
```
{
    "detail": [
        {
            "type": "missing",
            "loc": [
                "body",
                "font_type"
            ],
            "msg": "Field required",
            "input": null
        }
    ]
}
```

Reply:
Thank you for catching that. It seems the `font_type` parameter is missing in the form data when the download request is made. I'll ensure that the `font_type` is correctly included in the form data and address the issue. I'll push the fix shortly.